src/main/java/com/faforever/client/config/BaseConfig.java

```diff
@@ -22,6 +25,18 @@ MessageSource messageSource() {
 
   @Bean
   EventBus eventBus() {
-    return new EventBus((exception, context) -> exception.printStackTrace());
+    EventBus bus = new EventBus((exception, context) -> exception.printStackTrace());
+    bus.register(new DeadEventHandler());
+    return bus;
+  }
+
+  @Log
+  private static class DeadEventHandler {
+    @Subscribe
+    public void onDeadEvent(DeadEvent deadEvent) {
+      Object unhandledEvent = deadEvent.getEvent();
+      log.warning("No event handler registered for event of type " +
+          unhandledEvent.getClass().getSimpleName() + ": " + unhandledEvent);
```

Comment:
Cool, didn't know about DeadEvent. When you switch to slf4j, make sure to use playerholders `{}` instead of string concatenation and follow the [logging format guidelines](https://github.com/FAForever/java-guidelines/wiki/Coding-Standards#always-decorate-placeholders).